DSpace/DSpace
# Add a correction service to dspace to enhance the data quality (was Enrich local data via the OpenAIRE Graph)

## References
* Related to https://github.com/DSpace/RestContract/pull/181
* Related to https://github.com/DSpace/dspace-angular/pull/1562

## Description
The features included in this PR are the result of the OpenAIRE Call Innovation funded project "Enrich local data via the OpenAIRE Graph” awarded by 4Science (https://www.openaire.eu/open-call-winner-phase-1-4science). It provides a closer integration between DSpace and two OpenAIRE services, the Notification Broker and the OpenAIRE REST API.
Detailed documentation about the aims of the project, the implementation and the configuration options is available at https://4science.github.io/oaire-eld/#/
This PR regarding only the data correction section. The publication claim will be migrated with other PR.

## How to Use / Test

See frontend PR's description for details: https://github.com/DSpace/dspace-angular/pull/1562

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_
